kmod/core/core.c

```diff
@@ -681,11 +681,10 @@ static int kpatch_write_relocations(struct kpatch_module *kpmod,
      ( LINUX_VERSION_CODE >= KERNEL_VERSION(4, 4, 0) && \
       UTS_UBUNTU_RELEASE_ABI >= 7 ) \
     )
-               if (loc < core + kpmod->mod->core_layout.ro_size)
+               readonly = !!(loc < core + kpmod->mod->core_layout.ro_size);
 #else
-               if (loc < core + kpmod->mod->core_ro_size)
+               readonly = !!(loc < core + kpmod->mod->core_ro_size);
```

Comment:
The `!!` can be removed and it will still work and be slightly more readable IMO.